lib/elixir/lib/list.ex

```diff
@@ -246,11 +248,18 @@ defmodule List do
       iex> List.last([1, 2, 3])
       3
 
+      iex> List.last([1, 2, 3 | 4])
+      4
```

Comment:
Please add an empty line after this and before the `"""`.

Reply:
@whatyouhide this is a great coincidence. Do we remember I have fixed these exact thing in a PR in the past. Do you know which commit was that or PR? I have been looking for it for the past 20 minutes to no avail, and now you send this comment.

Reply:
I don't recall, sorry!

Reply:
well. I will use this comment for reference. thank you anyway